hydradx/tests/test_omnipool_amm.py

```diff
@@ -452,8 +452,14 @@ def test_remove_liquidity_no_fee(initial_state: oamm.OmnipoolState):
     initial_agent = Agent(
         holdings={token: 1000 for token in initial_state.asset_list + ['LRNA']},
     )
+    old_state, old_agent = initial_state.copy(), initial_agent.copy()
     # add LP shares to the pool
-    old_state, old_agent = oamm.simulate_add_liquidity(initial_state, initial_agent, 1000, i)
```

Comment:
Why is this changing? IMO testing simulate_add_liquidity is better, because you are testing against the "outermost" function.